circleci/templates/maven/config_template-no-docker-v1.yml

```diff
@@ -109,13 +116,47 @@ jobs:
       - image: circleci/ruby:2.4.1
     steps:
       - run: echo "*PLACEHOLDER* pretending to execute integration testing phase!"
-  Deploy Artifacts: # Deploy artifacts to Nexus or Maven Central
-    docker:
-      - image: circleci/ruby:2.4.1
-    steps:
-      - run: echo "*PLACEHOLDER* pretending to deploy build artifacts to Maven Central!"
 # END: UNDER CONSTRUCTION AREA ---------------------------------------
 
+  Tag Release:
+    docker: *image
+    steps:
+      # Tag Release creates a Git tag with given a given name and pushes it
+      # This job is governed by the following environment variables:
+      # $RELEASE - Set to true to perform a release, otherwise this step is skipped
+      # $RELEASE_TAG - The release tag for this version
+      # $RELEASE_VERSION - The version to release
+      # $NEXT_VERSION - The next development version
+      # To use this step the following MUST also be set: $GITHUB_KEY and $ION_GPG_KEYRING
+      - checkout
+      - run:
+          name: Tag Release
+          command: |
+            if [ $RELEASE = true ]
+            then
+              git checkout $CIRCLE_SHA1
+              git remote add ssh-origin git@github.com:connexta/$CIRCLE_PROJECT_REPONAME.git
+              echo ssh -i ${GITHUB_KEY} -l git -o StrictHostKeyChecking=no \\"\\$@\\" > run_ssh.sh
+              chmod +x run_ssh.sh
+              git push ssh-origin HEAD:$CIRCLE_BRANCH && git push ssh-origin $RELEASE_TAG
+            fi
+  Deploy Artifacts: # Deploy artifacts to Nexus or Maven Central
+    docker: *image
+    steps:
+      # Deploy Artifacts uses the Maven Deploy plugin and it's configuration from the pom.xml to deploy an artifact
+      # This job is governed by the following environment variables:
+      # $RELEASE - Set to true to perform a release, otherwise this step is skipped
+      # To use this step the following MUST also be set: $GITHUB_KEY and $ION_GPG_KEYRING
+      - run: *env
+      - checkout
+      - run:
+          name: Tag Release
```

Comment:
Didn't we tag the release above?
Seems like a redundant name... or I'm missing something obvious :-)

Reply:
Removed from above now